core/src/test/java/org/apache/accumulo/core/file/rfile/bcfile/CompressionTest.java

```diff
@@ -119,7 +126,17 @@ public void testSupport() {
     } catch (ClassNotFoundException e) {
       // that is okay
     }
+  }
 
+  @After
+  public void restoreSysProps() {
+    // Restore system property for next test
+    Algorithm.conf.clear();
+    if (extLz4 != null) {
+      System.setProperty(Compression.Algorithm.CONF_LZ4_CLASS, extLz4);
+    } else {
+      System.clearProperty(Compression.Algorithm.CONF_LZ4_CLASS);
+    }
```

Comment:
We run our unit tests in parallel and reuse JVM forks to do so. Messing with system properties like this could make these tests not thread safe. But, I'm not sure.